tests/main/classic-ubuntu-core-transition/task.yaml

```diff
@@ -29,8 +31,7 @@ execute: |
 
     . "$TESTSLIB/pkgdb.sh"
     echo "Ensure core is gone and we have ubuntu-core instead"
-    dpkg --purge --force-depends snapd
-    apt-get install -f -y
```

Comment:
apt-get install -f -y actually does dependency cleanup if you did a purge via dpkg --purge